strawberry/field.py

```diff
@@ -85,6 +87,29 @@ def __getattr__(self, attr):
     def __call__(self, *args, **kwargs):
         return self._wrapped_obj(self, *args, **kwargs)
 
+    def _get_permissions(self):
+        """
+        Gets all permissions defined in the permission classes
+        >>> strawberry.field(permission_classes=[IsAuthenticated])
+        """
+        if self.permission_classes:
+            return [permission() for permission in self.permission_classes]
+
+    def _check_permissions(self, info):
+        """
+        Checks if the permission should be accepted and
+        raises an exception if not
+        """
+        if not self._get_permissions():
+            return
+        for permission in self._get_permissions():
+            if not permission.has_permission(info):
+                message = getattr(permission, "message", None)
+                return self.permission_denied(message=message)
```

Comment:
can we inline the raise instead of calling another method?

Reply:
Alright will do